lib/LedgerSMB/Template/XLSX.pm

```diff
@@ -65,7 +65,8 @@ my $rowcount;
 my $currcol;
 
 sub _worksheet_handler {
-    $rowcount = -1;
+    $rowcount = 0;
+    -- $rowcount;
```

Comment:
@ehuelsmann if the vars are unnecessary, the subroutine is unnecessary, here (and in XLS.pm?).